content/developer/reference/user_interface/view_architectures.rst

```diff
@@ -2801,6 +2801,15 @@ sorted on the string of the field.
    graph view aggregations are performed on database content, non-stored
    function fields can not be used in graph views
 
+
+In Graph view a ``field`` can have a ``widget`` attribute to dictate its format.
+The widget should be a field formatter, of which the most interesting are
+``date``, ``datetime``, ``float_time``, and ``monetary``.
```

Comment:
you can remove date and datetime here (date/datetime fields are not used as measures)

Reply:
Hello 

Changes have been done